packages/wasm-dpp/test/unit/identity/Identity.spec.js

```diff
@@ -27,6 +38,7 @@ describe('Identity', () => {
           purpose: IdentityPublicKey.PURPOSES.AUTHENTICATION,
```

Comment:
Shouldn't we export constants from wasm as well?

Reply:
I used exports from rs-dpp, but it seems impossible to keep `IdentityPublicKey` namespace for the constants.